keps/sig-node/20190129-hugepages.md

```diff
@@ -194,23 +194,30 @@ status:
 
 #### Pod Specification
 
-A pod must make a request to consume pre-allocated huge pages using the resource
-`hugepages-<hugepagesize>` whose quantity is a positive amount of memory in
-bytes.  The specified amount must align with the `<hugepagesize>`; otherwise,
-the pod will fail validation.  For example, it would be valid to request
+Containers in a pod can cunsume huge pages by requesting pre-allocated
+huge pages. In order to request huge pages, A pod spec must specify a certain
+amount of huge pages using the resource `hugepages-<hugepagesize>` in container
+object. The quantity of huge pages must be a positive amount of memory in bytes.
+The specified amount must align with the `<hugepagesize>`; otherwise, the pod
+will fail validation. For example, it would be valid to request
 `hugepages-2Mi: 4Mi`, but invalid to request `hugepages-2Mi: 3Mi`.
 
 The request and limit for `hugepages-<hugepagesize>` must match.  Similar to
 memory, an application that requests `hugepages-<hugepagesize>` resource is at
 minimum in the `Burstable` QoS class.
 
+If multiple containers consume huge pages in the same pod, the request must be
```

Comment:
I think we would want to differentiate between init containers and regular containers, since the init containers are all guaranteed to be done when the regular containers start up.